main.js

```diff
@@ -31,7 +41,21 @@ const jitsiMeetWindowOptions = {
     height: 600,
     minWidth: 800,
     minHeight: 600,
-    titleBarStyle: 'hidden'
+    titleBarStyle: 'hidden',
+    webPreferences: { experimentalFeatures: true }
+};
+
+/**
+ * Options used when creating the micro mode window.
+ */
+const microWindowOptions = {
+    width: 320,
+    height: 240,
+    titleBarStyle: 'hidden',
+    frame: false,
+    alwaysOnTop: true,
+    x: 1570,
```

Comment:
are these absolute values for the position of the micro window?  wonder if we might get into trouble using absolute values here